proc/process.c

```diff
@@ -1295,10 +1298,11 @@ void proc_exit(int code)
 		kstack = current->kstack = current->execkstack;
 		kstack += current->kstacksz;
 
-		PUTONSTACK(kstack, int, FORKED);
-		PUTONSTACK(kstack, process_spawn_t *, spawn);
-		PUTONSTACK(kstack, thread_t *, current);
-		hal_jmp(proc_vforkedExit, kstack, NULL, 3);
+		hal_jmp(proc_vforkedExit, kstack, NULL, 3, (const arg_t[]) {
```

Comment:
As in process_exex, let's use intermediate var.
Also, this formatting doesn't work that good